relay-filter/src/web_crawlers.rs

```diff
@@ -93,6 +93,15 @@ mod tests {
             "pingdom",
             "lyticsbot",
             "AWS Security Scanner",
+
+            "Mozilla/5.0 (Linux; Android 6.0.1; Calypso AppCrawler Build/MMB30Y; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/53.0.2785.124 Mobile Safari/537.36",
+"Slackbot-LinkExpanding 1.0 (+https://api.slack.com/robots)",
+"Slack-ImgProxy 0.19 (+https://api.slack.com/robots)",
+"Slackbot 1.0(+https://api.slack.com/robots)",
+"Twitterbot/1.0",
+"FeedFetcher-Google; (+http://www.google.com/feedfetcher.html)",
```

Comment:
_nit_: Indent those lines properly, also below in `test_dont_filter_normal_user_agents`. Can you also split the long line?

Reply:
yeah that's what I don't get, I would expect cargo fmt to catch and correct that

Reply:
It's most likely because of the long line above. Rustfmt has had issues with strings that exceed the maximum length for quite a while. I think in recent versions, it skips formatting the immediate scope that the string is declared in. 

The workaround is to shorten the string or break it up into multiple lines.